code/tic-tac-tow-service/src/main/kotlin/pt/isel/daw/tictactow/domain/Board.kt

```diff
@@ -1,7 +1,7 @@
 package pt.isel.daw.tictactow.domain
 
 data class Board(
-    private val cells: Array<Array<State>>
+    val cells: Array<Array<State>>
```

Comment:
JSON serialisation logic is external, so cells need to be visible. What do you think of this?